protected/humhub/modules/file/messages/es/widgets_UploadInput.php

```diff
@@ -1,22 +1,6 @@
 <?php
-/**
```

Comment:
Why is deleted?

Reply:
This is an auto-generated message which isn't really needed and can be removed.

Reply:
auto-generated message if the file is new and was created. After removed the message you never see it again. 
That why to need restore it.

Reply:
Shouldn't matter once a pull from the translation site is made, originally the message wasn't included in the translation file. :thinking:

Reply:
According to my observations, the bot sends diff changes translation from lasted saved commit. Like `git pull`... Sometimes he pushes incorrect commit and rewrite GitHub translation, if it not was fixed. See https://github.com/humhub/humhub/issues/2679

Reply:
If it fixed, Bot will send only new translation row, not this message. If it not fixed,  Bot misses your changes and rewrite like my commit.
In any case, I recommend returning this message.

Reply:
Good point, once I'm on my work laptop I'll fix this. :smile:

Reply:
Should be fixed now. :+1: